src/app/containers/SwapFormContainer/index.tsx

```diff
@@ -75,8 +75,16 @@ export const SwapFormContainer: React.FC = () => {
       cancelDataRequest.current = c;
     });
     axios
-      .get(graphWrapperUrl[currentChainId] + 'cmc/summary?extra=true', {
-        cancelToken,
+      .get(graphWrapperUrl[currentChainId] + 'cmc/summary', {
+        params: {
+          extra: true,
+          stmp: Date.now(),
+        },
+        headers: {
+          'Cache-Control': 'no-cache',
+          Pragma: 'no-cache',
+          Expires: '30',
+        },
```

Comment:
Add this to getAssetData() call as well, or remove it from here - or is it intentional to only have headers set on this request?